.github/workflows/flaky-tests.yaml

```diff
@@ -0,0 +1,81 @@
+name: Flaky Tests Detector
+run-name: Flaky Tests Detector - ${{ github.run_id }} - @${{ github.actor }}
+
+on:
+  pull_request:
+    paths:
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+env:
+  ATTEMPTS: 30
+
+jobs:
+  test:
+    name: Flaky tests detector
+    runs-on: ubuntu-22.04-32core
+
+    permissions:
+      contents: read
+      packages: read
+
+    container:
+      image: ghcr.io/gravitational/teleport-buildbox:teleport12
```

Comment:
@r0mant Which tag should we use here? I don't think that we're publishing latest, but this is another place that we will need to update after a release.